examples/custom_bot.py

```diff
@@ -0,0 +1,127 @@
+import asyncio
+import logging
+import os
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+from .custom_context import MyContext
+
+__all__ = ('MyBot',)
+
+# Logging is the important part for all projects.
+# set the logger up for your own purposes.
+logger = logging.getLogger('bot')
```

Comment:
You need to get the `discord` logger for accessing the library's logging?

https://discordpy.readthedocs.io/en/latest/logging.html#logging-setup